core/src/avm2/globals/flash/system/security.rs

```diff
@@ -62,48 +44,11 @@ fn load_policy_file<'gc>(
     Ok(Value::Undefined)
 }
 
-fn show_settings<'gc>(
+pub fn showSettings<'gc>(
```

Comment:
I do prefer native methods to be `snake_case`, since eventually we plan to have a plenty amount of them.

Reply:
This is difficult to do properly, since there are methods that `navigateToURL` that would require special-casing to avoid  generating `navigate_to_u_r_l`.

Reply:
Actually, the `convert-case` crate seems to handle this correctly, so we could just add that as a dependency.